zksync_sdk/ethereum_signer/web3.py

```diff
@@ -21,3 +23,18 @@ def sign(self, message: bytes) -> TxEthSignature:
 
     def address(self) -> str:
         return self.account.address
+
+    def recovery_message(self, tx: TxEthSignature, message: bytes) -> str:
```

Comment:
Maybe it is better to be called `recover_signer` or `recover_signer_address`?

Also, why do we need to keep it as part of a Signer? It does not use `self`.

Reply:
Ok. I got you.